ext/date/tests/test-parse-from-format.phpt

```diff
@@ -54,6 +54,17 @@ object(DateTime)#3 (3) {
   string(6) "+02:00"
 }
 
+string(11) "Y-m-d H:i:s"
+string(19) "2008-07-08 22:14:12"
+object(DateTime)#3 (3) {
```

Comment:
Don't hard-code the object number. Instead do:
```suggestion
object(DateTime)#%d (3) {
```
(also for the other cases)

That requires the `--EXPECT--` above to be changed to `--EXPECTIF--`.